_data/simple-icons.json

```
@@ -5575,6 +5575,11 @@
             "hex": "261C29",
             "source": "http://mttr.net"
         },
+        {
+            "title": "Mautic",
+            "hex": "FDB933",
```

```suggestion
            "hex": "4E5E9E",
```

They seem to be using the purple more prominently. Not knowing the brand, I would assume that is the colour we should be using.

---

The first color mentioned in https://www.mautic.org/about/logos-and-graphics is blue (purplish). I agree this should be the color.

@adamrusted Feel free to commit your suggestion to merge this one!